test/functional/rpc_dumptxoutset.py

```diff
@@ -60,6 +62,16 @@ def run_test(self):
         assert_raises_rpc_error(
             -8, 'Invalid snapshot type "bogus" specified. Please specify "rollback" or "latest"', node.dumptxoutset, 'utxos.dat', "bogus")
 
+        self.log.info(f"Test that dumptxoutset failure does not leave the network activity suspended")
+        rev_file = node.chain_path / "blocks" / "rev00000.dat"
+        bogus_file = node.chain_path / "blocks" / "bogus.dat"
```

Comment:
Could use `node.blocks_path` instead of `node.chain_path` for these two lines.

Reply:
done